pages/api/webhooks/order-calculate-taxes.ts

```diff
@@ -7,32 +8,22 @@ import {
 import type { Handler } from "retes";
 import { toNextHandler } from "retes/adapter";
 import { Response } from "retes/response";
-import { calculateOrderTaxes } from "../../../backend/taxHandlers";
-import { OrderPayload } from "../../../backend/types";
-import {
-  getTaxJarConfig,
-  taxJarConfigIsValidToUse,
-} from "../../../backend/utils";
+import { calculateTaxesHandler } from "../../../backend/taxHandlers";
 
 const handler: Handler = async (request) => {
   const saleorDomain = request.headers[SALEOR_DOMAIN_HEADER];
 
-  const body: OrderPayload[] =
+  const payload: CalculateTaxesEventSubscriptionFragment =
     typeof request.body === "string" ? JSON.parse(request.body) : request.body;
 
-  const orderPayload: OrderPayload = body[0];
-
-  const taxJarConfig = await getTaxJarConfig(
-    saleorDomain,
-    orderPayload.channel.slug
-  );
-  const validData = taxJarConfigIsValidToUse(taxJarConfig);
-
-  if (!validData.isValid) {
-    return { body: validData.message, status: validData.status };
+  if (payload?.__typename === "CalculateTaxes") {
+    const taxObject = payload.taxBase!;
+    return await calculateTaxesHandler(taxObject, saleorDomain);
   }
-  const calculatedTaxes = await calculateOrderTaxes(orderPayload, taxJarConfig);
-  return Response.OK(calculatedTaxes.data);
+  return Response.BadRequest({
+    success: false,
+    message: "Incorrect payload event.",
+  });
 };
```

Comment:
Yes, I''ve extracted most of the bussiness logic and placed it inside the `backend` module. I didn't extract the logic related directly to the nextjs handler as I thought that `naxtjs` handler shouldn't be placed inside the backend module. 
Also, when app-sdk gives me a possibility to define list of events, instead of guessing, I will be able to merge these two endpoints into one: https://github.com/saleor/saleor-app-sdk/issues/26